src/models/Genre/parseGenre.ts

```diff
@@ -8,10 +8,10 @@ export default function parseGenre(entry: GenreContentfulEntry, getMovieData?: b
 		title: entry.fields.title as string,
 		movies: getMovieData
 			? ((entry.fields?.movies as MovieContentfulEntry[]) || []).map((movie) =>
-					parseMovieData(movie)
-			)
+					parseMovieSummary(movie)
```

Comment:
This is just a personal preference but ternaries like this are impossible to read IMO. You should just make a private function to handle this to make it easier to read.